src/main/java/htsjdk/samtools/util/AbstractLocusIterator.java

```diff
@@ -91,7 +91,8 @@
      * ii) any uncovered positions between the last AbstractLocusInfo and the first aligned base of the new read
      * have LocusInfos created and added to {@link #complete} if we are emitting uncovered loci
      */
-    final ArrayList<K> accumulator = new ArrayList<>(100);
+    //TODO: convert this to using a CircularArrayList so that the remove and random access operations will be fast.
```

Comment:
Can you make an issue out of this, rather than check in a TODO (unless you plan to do it in this PR)